### PR TITLE
Rm apns and SA endpoint stuff

### PR DESCRIPTION
**Pull request by techlab-lainio**
- APNS removed
- SA webhook endpoint type removed and related cmd helpers
- `pairwise` member variables removed from the cloud agent

### CODE REVIEW

File: cmd/agency.go [hunk 1]
```diff
@@ -24,7 +24,6 @@ Parent command for starting and pinging agency
 }
 
 var agencyStartEnvs = map[string]string{
-	"apns-p12-file":            "APNS_P12_FILE",
```
Comment (lauravuo-techlab) [inline cmd/agency.go:27]: Remove FCLI_AGENCY_APNS_P12_FILE from deployment Dockerfile as well

File: agent/apns/apns.go [hunk 2]
```diff
@@ -1,88 +0,0 @@
...
-import (
-	"crypto/tls"
-	"errors"
-	"os"
-
-	"github.com/findy-network/findy-agent/agent/psm"
-	"github.com/findy-network/findy-agent/agent/utils"
-	"github.com/golang/glog"
-	"github.com/lainio/err2"
-	"github.com/sideshow/apns2"
```
Comment (lauravuo-techlab) [inline agent/apns/apns.go:12]: Update module files as these are not needed anymore

File: cmd/agency.go [hunk 3]
```diff
@@ -132,7 +131,6 @@ func init() {
 	aCmd.VersionInfo = "findy-agent v" + utils.Version
 
 	flags := startAgencyCmd.Flags()
-	flags.StringVar(&aCmd.APNSP12CertFile, "apns-p12-file", "", flagInfo("APNS certificate p12 file", AgencyCmd.Name(), agencyStartEnvs["apns-p12-file"]))
```
Comment (lauravuo-techlab) [inline cmd/agency.go:135]: Unrelated to this PR, but came to my mind... Should we remove also the grpc-flag, as it is now the only way to command agent and should be enabled always?

**Review by lauravuo-techlab — APPROVED** [review]
Looks good! Two minor comments.

File: cmd/agency.go (hunk 1 above)
Comment (techlab-lainio) [reply cmd/agency.go:27]: yes, I actually thought this but was so lazy that didn't want to touch those Docker related files. Good point, I'll fix this!

File: cmd/agency.go (hunk 3 above)
Comment (techlab-lainio) [reply cmd/agency.go:135]: Excellent point! Will fix it in this same PR.

**Pull request merged**